src/main/java/uk/ac/cam/cl/dtg/isaac/api/AssignmentFacade.java

```diff
@@ -796,7 +795,7 @@ private ArrayList<String[]> buildGroupAssignmentsReportBody(
 
     for (RegisteredUserDTO groupMember : groupMembers) {
       // FIXME Some room for improvement here, as we can retrieve all the users with a single query.
-      // FIXME Not urgent, as the dominating query is the one that retrieves question attempts above.
+      //  Not urgent, as the dominating query is the one that retrieves question attempts above.
```

Comment:
Two spaces after `//`, barely unreadable 😝

Reply:
Now that I'm seeing this in the IDE, I think I get it: you've indented it so that this is also part of the FIXME and it's highlighted by the IDE. If that's the case, then please ignore all my other comments regarding whitespace.

Reply:
I don't know why but the extra space causes the analysis tool and other indicators to recognise the second line as a continuation of the same comment rather than cutting it off. Might be something IDE-specific though, so I suppose I could undo the changes if you really want?